nano/node/messages.cpp

```diff
@@ -671,6 +672,11 @@ std::size_t nano::confirm_ack::size (const nano::message_header & header)
 	return nano::vote::size (count);
 }
 
+bool nano::confirm_ack::is_rebroadcasted () const
```

Comment:
Since this field is currently filled with 0 and we're trying to determine if the message comes from the originator, should this be an is_originator () flag that's set to 1? Otherwise we need to also check the channel's version number.

Reply:
So, my thinking was that, since we want to ignore rebroadcasted votes in rep crawler, to treat messages without this flag set (ie. old versions) as originators by default. Isn't this what we want?

Reply:
That's a good point, let's keep it how you have it.